src/common/test-jsonfds.c

```diff
@@ -422,10 +422,10 @@ test_memfd_error_cases (void)
 
       content = cockpit_memfd_read (fd, &error);
       cockpit_assert_error_matches (error, G_FILE_ERROR, G_FILE_ERROR_INVAL, "*not memfd?*");
+      g_assert (content == NULL);
       g_clear_error (&error);
       close (fd);
 
-
```

Comment:
I was using the extra space to separate conceptually separate independent "cases".

Reply:
ok, I put them back.